estuary/api/v1.py

```diff
@@ -240,14 +242,21 @@ def get_siblings(resource, uid):
     :raises NotFound: if the item is not found
     :raises ValidationError: if an invalid resource was requested
     """
+    story_type = request.args.get('story_type', 'container').lower()
     # This is the node that is part of a story which has the relationship to the desired
     # sibling nodes
     story_node = get_neo4j_node(resource, uid)
     if not story_node:
         raise NotFound('This item does not exist')
 
-    # Defaulting to Container Story until we modify this endpoint to suit the module changes
-    story_manager = ContainerStoryManager()
+    for class_name in current_app.config['STORY_MANAGER_SEQUENCE']:
+        if class_name.lower().startswith(story_type):
```

Comment:
Optional: I'm realizing now that this logic is fine for now but if we added another story path like `ModuleContainerStoryManager`, this would cause problems.

Reply:
In such case, I think we can then have a dict of valid parameters and their corresponding classes. Also the parameter in such a case would have to be something like 'module-container' or something similar. I think we can handle that using this logic too.

Reply:
@yashvardhannanavati a dictionary would be a good option. You can leave it as is or implement that.